docs/api/ratsd.yaml

```diff
@@ -0,0 +1,148 @@
+openapi: 3.0.0
+info:
+  title: RATS Evidence Collection Daemon
+  version: 0.0.1
+tags: []
+paths:
+  /ratsd/chares:
```

Comment:
```suggestion
  /ratsd/chalrsp:
```

Reply:
An alternative is to use Greek in the URI (`/ratsd/Χάρης`) and the media type (`application/vnd.veraison.Χάρης+json`):

```http
POST /ratsd/Χάρης HTTP/1.1
Host: host.example
Accept: application/eat+jwt; eat_profile="tag:github.com,2024:veraison/ratsd"
Content-Type: application/vnd.veraison.Χάρης+json
Content-Length: 33

{
  "nonce": "MIDBNH28iioisjPy"
}
```

A fun conversation to have with IETF's Internationalization Directorate :-)